### PR TITLE
fix: Remove usefullinks section and make question section weider on large screen

### PR DESCRIPTION
**Pull request by yehia67**
Fix Issue https://github.com/freeCodeCamp/freeCodeCamp/issues/35397   . I  removed hepful-links col and  row then I made question class more wider for large screen

### CODE REVIEW

**Comment by ahmadabdolsaheb** [comment]
@yehia67 , there seems to be a problem in /home/travis/build/freeCodeCamp/freeCodeCamp/client/src/pages/about.js
  182:134  error  Parsing error: Unterminated string constant

File: client/src/pages/about.js [hunk 1]
```diff
@@ -3,337 +3,219 @@ import Helmet from 'react-helmet';
...
-  faMedium,
-  faFacebook
+    faLinkedin,
+    faGithub,
+    faTwitter,
+    faYoutube,
+    faMedium,
+    faFacebook
 } from '@fortawesome/free-brands-svg-icons';
 import {
```
Comment (ahmadabdolsaheb) [inline client/src/pages/about.js]: or these, anymore :)

File: client/src/pages/about.js [hunk 2]
```diff
@@ -3,337 +3,219 @@ import Helmet from 'react-helmet';
 import { Grid, Col, Row, Table } from '@freecodecamp/react-bootstrap';
 import FontAwesomeIcon from '@fortawesome/react-fontawesome';
 import {
```
Comment (ahmadabdolsaheb) [inline client/src/pages/about.js]: or these

File: client/src/pages/about.js [hunk 3]
```diff
@@ -3,337 +3,219 @@ import Helmet from 'react-helmet';
 import { Grid, Col, Row, Table } from '@freecodecamp/react-bootstrap';
 import FontAwesomeIcon from '@fortawesome/react-fontawesome';
```
Comment (ahmadabdolsaheb) [inline client/src/pages/about.js]: I don't think we need these anymore.

**Comment by yehia67** [comment]
Okay thanks. I Will fix it.

**Comment by ahmadabdolsaheb** [comment]
@yehia67, see where it says all checks have failed, it the next line click on details. in the next page see where they have listed the errors in red. click on it. on the next page scroll until you find the errors in listed in red. it is a lint error. so it should be an easy fix.

**Comment by yehia67** [comment]
Yea, I understand. Unfortunately, I have lecture now I will resolve it right after The lecture. Thanks and sorry for bothering you.

**Comment by thecodingaviator** [comment]
@ahmadabdolsaheb @RandellDawson I'm wondering if all the changes would actually work. The formatting of the tag declaration and closing is really messed up i.e. somewhere it is like 
```html
<
 h4>
``` 
somewhere it is like `< /p>`

**Comment by Ensive** [comment]
@yehia67 
Hi, let me know if you need help with this. I could contribute directly to your PR.

**Comment by yehia67** [comment]
Yes, I have problem in running freecodecamp. Some dependencies didn't installed  yet after running `npm install  `  
```
npm WARN lifecycle @freecodecamp/freecodecamp@0.0.1~postinstall: cannot run in wd @freecodecamp/freecodecamp@0.0.1 npm run bootstrap (wd=/home/yehia/Desktop/yehia/Projects/web/freeCodeCamp)
npm WARN acorn-jsx@5.0.1 requires a peer of acorn@^6.0.0 but none is installed. You must install peer dependencies yourself.
```

So I am trying to install them manually.

**Comment by Bouncey** [comment]
@yehia67 I found this when trying to `npm install` as the root user, or as a user that does not have write permission in the current directory.

What os are you using?

**Comment by yehia67** [comment]
I am using Ubuntu 18.04. I found  this line.
```
lerna ERR! npm install stderr:
ERR! sharp EACCES: permission denied, mkdir '/root/.npm'
ERR! sharp Please see http://sharp.pixelplumbing.com/page/install
gyp WARN EACCES user "root" does not have permission to access the dev dir "/root/.node-gyp/10.15.1"

```
I had searched for this 10.15.1 folder and I found it' permission for user-501 (I don't know what it is) However I had change It to root permission. And I will keep you with Updates.

**Comment by yehia67** [comment]
Unfortunately, I keep give me the same warnings 
image

**Comment by raisedadead** [comment]
Hi @yehia67 

Thanks for contributing to freeCodeCamp.

Please reachout to [Contributors Chat room](https://gitter.im/FreeCodeCamp/Contributors), the awesome folks their will be able to walk you through the troubleshooting. 

Keep this PR discussion for the scope of the original issue.

**Comment by yehia67** [comment]
Okay Thanks @raisedadead

**Pull request closed without merging**